frontend/test/metabase/scenarios/sharing/alert/alert.cy.spec.js

```diff
@@ -207,3 +207,49 @@ describe("scenarios > alert", () => {
     });
   });
 });
+
+describeWithToken("scenarios > alert (EE)", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should validate for approved email domains", () => {
+    const email = "mailer@metabase.test";
+    const domain = "metabase.example";
+
+    setupDummySMTP();
+    setAllowedEmailDomains(domain);
+
+    cy.createQuestion({
+      name: "Orders",
+      query: { "source-table": ORDERS_ID },
+    }).then(({ body: { id: questionId } }) => {
+      cy.visit(`/question/${questionId}`);
+    });
+
+    cy.icon("bell").click();
+    cy.findByText("Set up an alert").click();
+    addEmailRecipient(email);
```

Comment:
There is already a recipient pre-selected. That' an admin (Bobby Tables). His name is `admin@metabase.test`.
If I remove the explicit email recipient from this line and leave only Bobby Tables, the alert gets successfully created.

That seems like a bug to me.